sdk/go/auto/local_workspace.go

```diff
@@ -160,13 +160,16 @@ func (l *LocalWorkspace) SetConfig(ctx context.Context, stackName string, key st
 	if val.Secret {
 		secretArg = "--secret"
 	}
-	pathArg := ""
+
+	var args []string
 	if val.IsPath {
-		pathArg = "--path"
+		args = []string{"config", "set", secretArg, "--path", key, "--stack", stackName,
```

Comment:
We need to ensure that the values of `args` match except for path. I would like to see this done without duplicating the entire arg list. Every time I look at this, I need to manually verify that the only difference between `val.IsPath` and `!val.IsPath` is the extra `"--path"` in the arg list.